src/SkyApm.Core/Tracing/SegmentContextFactory.cs

```diff
@@ -212,7 +212,7 @@ private SegmentContext GetParentSegmentContext(SpanType spanType)
                 case SpanType.Entry:
                     return null;
                 case SpanType.Local:
-                    //return _entrySegmentContextAccessor.Context;
+                    return _entrySegmentContextAccessor.Context;
```

Comment:
@yang-xiaodong @Ahoo-Wang You need to discuss about this line.

Reply:
@Ahoo-Wang Can you explain the reason why commented?

Reply:
I can't...

Reply:
@Ahoo-Wang The change was made in this PR. Do you remember it?
https://github.com/SkyAPM/SkyAPM-dotnet/pull/180/files#diff-f8ba2312c7af27874777d8416b239dadR215

Reply:
@Ahoo-Wang Please confirm that this change has no effect on you

Reply:
@ElderJames No response of Ahoo-Wang,  I think here should not be commented

Reply:
@yang-xiaodong He would check it before tonight.

Reply:
@yang-xiaodong @ElderJames 
I'm really sorry, I haven't had time to check, I have to have a full day of meetings tomorrow, I've been too busy lately. If you have any patience, I will check at the end of the meeting tomorrow, I hope the meeting will end earlier tomorrow.

Reply:
@yang-xiaodong @ElderJames 
I checked this line of code changes and didn't seem to have any negative impact.

Reply:
@Ahoo-Wang Fine, thanks for check.

Reply:
@ElderJames @yang-xiaodong @Ahoo-Wang This line should be comment. The parent of a local span may be a local span. If uncomment this line of code, the parent of the local span can only be entry span.
```csharp
[HttpGet]
[Route("/test")]
public async Task<string> Get()
{
    var parentLocalSegment = _tracingContext.CreateLocalSegmentContext("parent");

    var childLocalSegment = _tracingContext.CreateLocalSegmentContext("child");
    _tracingContext.Release(childLocalSegment);

    _tracingContext.Release(parentLocalSegment);

    return "ok";
}
```
current tree view:
![image](https://user-images.githubusercontent.com/15099549/164366970-5a073f25-05ed-4127-9020-899d2fd7b7b4.png)

after comment the code:
![image](https://user-images.githubusercontent.com/15099549/164367132-24ec9d82-521b-49b7-8722-5ba1fe33309f.png)